frontend/src/components/GoalForm/Form.js

```diff
@@ -188,7 +188,7 @@ export default function Form({
         />
       ))}
 
-      { (status !== 'Closed' || userCanEdit) && (
```

Comment:
I think this needs to be adjusted so that users without editing privileges don't see the button. (View a goal in a region where you only have read permissions, not write, to reproduce)

![Screen Shot 2022-11-17 at 9 01 58 AM](https://user-images.githubusercontent.com/3288586/202466603-3a6a1975-cbf3-4c4e-ba2a-3ad5df3d37e2.png)

Reply:
This has been fixed to be an && (nice catch).